src/renderer/components/LoginComponent/LoginComponent.tsx

```diff
@@ -1,32 +1,55 @@
-import React from 'react';
-import { Typography } from '@mui/material';
-import { FaGithub } from 'react-icons/fa';
-import { useDispatch } from 'react-redux';
-import Button from '@mui/material/Button';
-import { authorizeGitHubUserAsync } from '../../../shared/redux/slices/currentUserSlice';
-import './LoginComponent.scss';
-
-const LoginComponent = () => {
-  const dispatch = useDispatch();
-  return (
-    <div className='container'>
-      <Typography fontWeight='normal' fontStyle='italic' color='text.secondary'>
-        Welcome to VNlab
-      </Typography>
-      <div>
-        <Button
-          variant='contained'
-          className='button'
-          color='green'
-          onClick={() => {
-            dispatch(authorizeGitHubUserAsync(false));
-          }}
-        >
-          Log in with GitHub <FaGithub />
-        </Button>
-      </div>
-    </div>
-  );
-};
-
-export default LoginComponent;
+import React from 'react';
+import { Typography } from '@mui/material';
+import { FaGithub } from 'react-icons/fa';
+import { useDispatch } from 'react-redux';
+import Button from '@mui/material/Button';
+import { authorizeGitHubUserAsync } from '../../../shared/redux/slices/currentUserSlice';
+import './LoginComponent.scss';
+import logo from '../../assets/publab.png';
+
+const LoginComponent = () => {
+  const dispatch = useDispatch();
+  return (
+    <div className='background'>
+      <div className='language-box'>
+        <Button variant='text' className='language'>
+          {' '}
+          <p>English</p>{' '}
+        </Button>
+        <Button variant='text' className='language'>
+          {' '}
+          <p>Polski</p>{' '}
+        </Button>

```

Comment:
You can use the `supportedLocales` object to render the buttons with code along the lines of this:

```tsx
const LoginComponent = () => {
  const dispatch = useDispatch();
  const { t } = useTranslation();

  return (
    <div className='background'>
      <div className='language-box'>
        {supportedLocales.map((locale) => (
          <Button key={locale} variant='text' className='language'>
            <p>{t(`AppSettings.language.${locale}`)}</p>
          </Button>
        ))}
      </div>
      // ... other code
    </div>
  );
};
```

This way it will render all possible locales that the app has as it could change in the future. This will also already intorduce a bit of internationalisation to the view.